src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -30,6 +32,8 @@ namespace SonarLint.VisualStudio.TypeScript.EslintBridgeClient
 {
     internal interface IEslintBridgeClient : IDisposable
     {
+        Task InitLinter(IEnumerable<Rule> rules);
```

Comment:
It would be useful to have a link to the corresponding Java and JS implementations  at the top of this file.